docker-compose/README.adoc

```diff
@@ -0,0 +1,53 @@
+ifdef::env-github[]
+:status:
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:table-caption!:
+:namespace: mobile-security-service
+endif::[]
+
+:toc:
+:toc-placement!:
+
+= UnifiedPush Server - Docker Compose
+
+== Using Docker-Compose
+
+For your convenience, we do have an easy way of launch different configurations of the UPS, using our link:./docker-compose[Docker compose files]
+
+By the following command you can startup this project locally using Docker-Compose.
+
+[source,shell]
+----
+docker-compose -f docker-compose.yaml up
+----
+
+IMPORTANT: There is a bug on docker-compose for Mac with does not expose internal docker IP. To fix it you will need following the steps in <<Solving bug in Mac OS to use Docker-Compose>>
+
+== Solving bug in Mac OS to use Docker-Compose
+
+. Expose the Keycloak port in the link:docker-compose.yaml[docker-compose.yaml]
```

Comment:
This step is not needed anymore and should be excluded. We already have added this in the commited `docker-compose.yaml` See: https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/docker-compose/docker-compose.yaml#L50-L51

Reply:
Now ti makes more sense. Really tks :-)
Done 👍